mce-dbus.c

```diff
@@ -1276,6 +1302,29 @@ peerinfo_set_owner_name(peerinfo_t *self, const char *name)
     return changed;
 }
 
+pid_t
+peerinfo_get_proxy_pid(const peerinfo_t *self)
+{
+    return self->pi_proxy_pid;
+}
+
+static void
+peerinfo_set_proxy_pid(peerinfo_t *self, pid_t pid)
+{
+    if( self->pi_proxy_pid == pid )
+	goto EXIT;
```

Comment:
There is a weird mix of indentation with spaces and tabs in this change. For example this line is indented with one tab and line above has four spaces but I'm not sure what was the intention.

Reply:
It is just what "tab=8 indent=4" looked like until people started messing with tab sizes. So, in context of this particular file it is intentional.

Reply:
Okay :sweat_smile: